time stamp
float64 exposure_time
float64 bias
int32 threshold
string interface
int16[65536] image
